src/App.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import Products from "./modules/Products";
+import Orders from "./modules/Orders";
+
+const App = () => {
+  const [products, setProducts] = useState([]);
+  const [order, setOrder] = useState();
+
+  useEffect(() => {
+    Products.index().then((data) => {
+      setProducts(data);
+    });
+  }, []);
+
+  const addToOrder = (product_id) => {
+    Orders.create(product_id, 1).then((response) => {
+      debugger;
```

Comment:
Don't need to say too much about this one!

Reply:
It's gone!